src/views/RecIM/components/Forms/ActivityForm/index.jsx

```diff
@@ -168,9 +168,9 @@ const ActivityForm = ({
         registrationStart: activity.RegistrationStart,
         registrationEnd: activity.RegistrationEnd,
         typeID:
-          activityTypes.find((type) => type.ID === activity.TypeID) == null
+          activityTypes.find((type) => type.Description === activity.Type) == null
             ? ''
-            : activityTypes.find((type) => type.ID === activity.TypeID).Description,
+            : activityTypes.find((type) => type.Description === activity.Type).Description,
```

Comment:
Firstly, you can make this more efficient:
```suggestion
          activityTypes.find((type) => type.Description === activity.Type)?.Description ?? '',
```

But secondly, do you need to search all activityTypes? Can't you just use `activity.Type` directly now?
```suggestion
	          activity.Type,
```

Reply:
@EjPlatzer This is actually a reverse null coalescence. I want it to be an empty string **if** it's null

Reply:
> @EjPlatzer This is actually a reverse null coalescence. I want it to be an empty string **if** it's null

```javascript
activity.Type ?? ''
```

Reply:
Actually no you're right, I think this was the workaround for the janky ID logic I had before